tests/adapter/.gitrepo

```diff
@@ -0,0 +1,12 @@
+; DO NOT EDIT (unless you know what you are doing)
+;
+; This subdirectory is a git "subrepo", and this file is maintained by the
+; git-subrepo command. See https://github.com/git-commands/git-subrepo#readme
```

Comment:
We can update the code here and sync it back to dbt-tests-adapter via `git subrepo push tests/adapter`.

Right now I'm pushing and pulling commits to dbt-tests-adapter. We'll need to lock that down and set up a workflow. I'm thinking if a pull request touches these files, we'll do a git push at that point. We should be able to handle pull request in dbt-tests-adapter also. I think it might be pretty confusing to have our internal adapter repositories able to push/pull also. We can discuss what process would work, but first inclination is to limit it to dbt-core plus dbt-tests-adapter pull requests.

Reply:
I share your inclination. It's highly convenient that we can make changes in this repo, to coincide with `dbt-core` changes, and then push them to the external repo. We don't need that ability for plugins; they should open a PR in `dbt-core` or `dbt-tests-adapter`

Reply:
Since we're not using subrepo anymore, the dbt.tests.adapter code will live just in the dbt-core repo. We can deal with packaging for external users at a later point.